pkg/pagerduty/pagerduty.go

```diff
@@ -88,7 +115,7 @@ func (p PagerDuty) AcknowledgeIncident(incident Incident) error {
 func (p PagerDuty) manageIncident(o []sdk.ManageIncidentsOptions) error {
 	ctx, cancel := context.WithTimeout(context.Background(), pagerDutyTimeout)
 	defer cancel()
-	_, err := p.c.ManageIncidentsWithContext(ctx, p.userEmail, o)
+	_, err := p.c.ManageIncidentsWithContext(ctx, p.currentUser.Email, o)
```

Comment:
since we no longer will have current user we can actually just user the p.UserEmail from the sdkClient.